README.rst

```diff
@@ -154,6 +154,14 @@ Filter manga chapters by language and chapter number:
     $ gallery-dl "r:https://pastebin.com/raw/FLwrCYsT"
 
 
+If a sites address is nonstandard for its extractor you can prefix the URL with the 
```

Comment:
```suggestion
If a site's address is nonstandard for its extractor, you can prefix the URL with the 
```

Sorry for being nit-picky, but I think it should be `site's` instead of `sites`, and the comma also seems to belong there.